yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -0,0 +1,229 @@
+	db, err := sql.Open("sqlite3", fmt.Sprintf("%s%s", GetDBFilePath(), metadb.SQLITE_OPTIONS))
+	if err != nil {
+		return nil, fmt.Errorf("error opening assessment db %s: %w", GetDBFilePath(), err)
+	}
+
+	return &AssessmentDB{db: db}, nil
+}
+
+func (adb *AssessmentDB) BulkInsert(table string, records [][]string) error {
+	tx, err := adb.db.Begin()
+	if err != nil {
+		return fmt.Errorf("error starting transaction for bulk insert into %s: %w", table, err)
+	}
+
+	columnNames := records[0]
+	stmtStr := fmt.Sprintf(`INSERT INTO %s (%s) VALUES (%s)`, table,
+		strings.Join(columnNames, ", "), strings.Repeat("?, ", len(columnNames)-1)+"?")
+
+	stmt, err := tx.Prepare(stmtStr)
+	if err != nil {
+		return fmt.Errorf("error preparing statement for bulk insert into %s: %w", table, err)
+	}
+
+	for rowNum := 1; rowNum < len(records); rowNum++ {
+		row := utils.ConvertStringSliceToInterface(records[rowNum])
+		_, err = stmt.Exec(row...)
+		if err != nil {
+			return fmt.Errorf("error inserting record for bulk insert into %s: %w", table, err)
+		}
+	}
+
+	err = tx.Commit()
+	if err != nil {
+		return fmt.Errorf("error committing transaction for bulk insert into %s: %w", table, err)
+	}
+
+	return nil
+}
+
+// populate migration_assessment_stats table using the data from other tables
+func (adb *AssessmentDB) PopulateMigrationAssessmentStats() error {
+	INSERT_TABLE_STATS := fmt.Sprintf(`INSERT INTO %s (schema_name, object_name, row_count, reads, writes, isIndex, parent_table_name, size)
+	SELECT
+		trc.schema_name,
+		trc.table_name AS object_name,
+		trc.row_count,
+		tio.seq_reads as reads,
+		tio.row_writes as writes,
+		0 AS isIndex,
+		NULL AS parent_table_name, 
+		ts.size
+	FROM table_row_counts trc
+	LEFT JOIN table_index_iops tio ON trc.schema_name = tio.schema_name AND trc.table_name = tio.object_name
+	LEFT JOIN table_index_sizes ts ON trc.schema_name = ts.schema_name AND trc.table_name = ts.object_name;`, MIGRATION_ASSESSMENT_STATS)
+
+	INSERT_INDEX_STATS := fmt.Sprintf(`INSERT INTO %s (schema_name, object_name, row_count, reads, writes, isIndex, parent_table_name, size)
+	SELECT
+		itm.index_schema AS schema_name,
+		itm.index_name AS object_name,
+		NULL AS row_count,
+		tio.seq_reads as reads,
+		tio.row_writes as writes,
+		1 AS isIndex,
+		itm.table_schema || '.' || itm.table_name AS parent_table_name,
+		ts.size
+	FROM index_to_table_mapping itm
+	LEFT JOIN table_index_iops tio ON itm.index_schema = tio.schema_name AND itm.index_name = tio.object_name
+	LEFT JOIN table_index_sizes ts ON itm.index_schema = ts.schema_name AND itm.index_name = ts.object_name;`, MIGRATION_ASSESSMENT_STATS)
+
+	_, err := adb.db.Exec(INSERT_TABLE_STATS)
+	if err != nil {
+		return fmt.Errorf("error executing INSERT_TABLE_STATS on %s table: %w", MIGRATION_ASSESSMENT_STATS, err)
+	}
+
+	_, err = adb.db.Exec(INSERT_INDEX_STATS)
+	if err != nil {
+		return fmt.Errorf("error executing INSERT_INDEX_STATS on %s table: %w", MIGRATION_ASSESSMENT_STATS, err)
+	}
```

Comment:
Make an array of statements and Exec them in a loop.

Reply:
done

Reply:
dne